post.hbs

```diff
@@ -64,32 +64,24 @@
             </section> }}
             </footer>
         </article>
-
-        <aside class="gh-readmore">
-            <div class="gh-readmore-inner w-full max-w-screen-xl mx-auto my-0">
-                <div class="gh-readmore-next">
-                    {{#next_post}}
-                        <a href="{{url}}">
-                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
-                            &nbsp;
-                            <h4>
-                                {{title}}
-                            </h4>
-                        </a>
-                    {{/next_post}}
-                </div>
-                <div class="gh-readmore-prev">
-                    {{#prev_post}}
-                        <a href="{{url}}">
-                            <h4>
-                                {{title}}
-                            </h4>
-                            &nbsp;
-                            <i class="fa fa-chevron-right" aria-hidden="true"></i>
-                        </a>
-                    {{/prev_post}}
-                </div>
-            </div>
+        
+        {{!-- implementation of related posts --}}
+        <article>
+            <br> {{!-- spacing --}}
+            <hr> {{!-- divider --}}
+            <br> 
+            <h2>
+                Related Articles
+            </h2> {{!-- title --}}
+            <br>
+            <div id="related-articles" class="grid grid-cols-1 lg:grid-cols-3 gap-x-20 gap-y-10 m-auto">
+                {{#get "posts" include="tags" limit="3" filter="tags:{{primary_tag.slug}}+id:-{{id}}" order="title asc" as |related|}}
+                    {{#foreach related}}
+                        <div class="row">{{> "story-postcard"}}</div>
+                    {{/foreach}}
```

Comment:
Let's also add an `else` clause that will show a placeholder message if there are no related posts.

Reply:
Done